cmd/res/profiles/llrp.device.profile.yaml

```diff
@@ -0,0 +1,154 @@
+name: "LLRP-Device-Profile"
+manufacturer: "Intel"
+model: "RFID Reader"
+labels:
+  - "LLRP"
+  - "RFID"
+description: "A generic device profile for readers that communicate over LLRP"
+
+deviceResources:
+  - 
+    name: "ReaderCapabilities"
+    description: "The capabilities of a reader"
+    properties:
+      valueType: "Object"
+      readWrite: "R"
+
+  - 
+    name: "ReaderConfig"
+    description: "Current configuration info of a reader"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "ROSpecID"
+    description: "Client-generated Reader Operation Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+
+  - 
+    name: "AccessSpecID"
+    description: "Client-generated Access Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+
+  - 
+    name: "ROSpec"
+    description: >-
+      An ROSpec is a "Reader Operation Specification", i.e.,
+      a declarative description of when and how
+      a reader should turn on its antennas and report data.
+      It's passed via EdgeX as a JSON object.
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "AccessSpec"
+    description: ""
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+      
+  - 
+    name: "ROAccessReport"
+    description: >-
+      An ROAccessReport is the main report type in LLRP.
+      They contain the TagReportData or RFSurveyReportData,
+      as well as any Custom parameter data.
+    properties:
+      valueType: "Object" # not actually readable; it's async
+      readWrite: "R"
+
+  - 
+    name: "ReaderEventNotification"
+    description: >-
+      Readers generate Reader Event Notifications for a variety of events,
+      including changes in connectivity, antenna additions/removals,
+      GPIO state changes, new report data, and exception events.
+      Some event notifications are configurable.
+    properties:
+      valueType: "Object" # not actually readable; it's async
+      readWrite: "R"
+
+  - 
+    name: "Action"
+    description: >-
+      EdgeX's Device Service SDK only distinguishes between "GET" and "PUT",
+      and all PUT parameters must be defined deviceResources.
+      Since we won't have, e.g., information about the HTTP path used in a request,
+      this resource exists only to distinguish actions to take on other resources.
+    properties:
+      valueType: "String"
+      readWrite: "W"
+
+deviceCommands:
+
+  - 
+    name: enableROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Enable" }
+
+  - 
+    name: startROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
```

Comment:
That is out of our knowledge. If you think so, then yes.

Reply:
I looked at the old profile, and from what I can tell, the `defaultValue` was `0` there as well. I'm not sure if it matters, since we expect the user to always send a value. @saites any thoughts on this?

Reply:
The defaultValue should be removed then, requiring it to be sent.

Reply:
I don't know if there are significant differences between this and the old way (using [`parameter`](https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/cmd/res/llrp.device.profile.yaml#L93)), but they were given defaults because LLRP often treats `0` as a special ID to target all objects of a given type. In other words, sending `EnableROSpec` with an `ROSpecID` of `0` tells the Reader it should enable all of the `ROSpec`s currently on the device.

Incidentally, this is why they were left out of the `Delete` commands -- we didn't want to default to deleting everything if you accidentally forgot to specify the value.

Reply:
the default values have been removed